lib/datasource/deb/index.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { GetPkgReleasesConfig, getPkgReleases } from '..';
+import { DebLanguageConfig } from './types';
+// import * as httpMock from '../../../test/http-mock';
```

Comment:
Tests need to mock any web requests, so you should try to create a minimal packages file

Reply:
That makes sense to me. Is it okay to store a compressed file within this repository and serve it with nock's `replyWithFile` parameter? Or should the content be "clear" text within the source code and be compressed on the fly?

Reply:
Can be a compressed file, but should be as small as possible. But you can also compress on the fly if it's fast.

Reply:
Done